cluster-provision/k8s/1.19/provision.sh

```diff
@@ -0,0 +1,318 @@
+kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f "$cni_manifest"
+
+# Wait at least for 7 pods
+while [[ "$(kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system --no-headers | wc -l)" -lt 7 ]]; do
+    echo "Waiting for at least 7 pods to appear ..."
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system
+    sleep 10
+done
+
+# Wait until k8s pods are running
+while [ -n "$(kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system --no-headers | grep -v Running)" ]; do
+    echo "Waiting for k8s pods to enter the Running state ..."
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system --no-headers | >&2 grep -v Running || true
+    sleep 10
+done
+
+# Make sure all containers are ready
+while [ -n "$(kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system -o'custom-columns=status:status.containerStatuses[*].ready,metadata:metadata.name' --no-headers | grep false)" ]; do
+    echo "Waiting for all containers to become ready ..."
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system -o'custom-columns=status:status.containerStatuses[*].ready,metadata:metadata.name' --no-headers
+    sleep 10
+done
+
+kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system
+
+kubeadm reset --force
+
+# Create local-volume directories
+for i in {1..10}
+do
+  mkdir -p /var/local/kubevirt-storage/local-volume/disk${i}
+  mkdir -p /mnt/local-storage/local/disk${i}
+  echo "/var/local/kubevirt-storage/local-volume/disk${i} /mnt/local-storage/local/disk${i} none defaults,bind 0 0" >> /etc/fstab
+done
+chmod -R 777 /var/local/kubevirt-storage/local-volume
+
+# Setup selinux permissions to local volume directories.
+chcon -R unconfined_u:object_r:svirt_sandbox_file_t:s0 /mnt/local-storage/
+
+# Pre pull fluentd image used in logging
+docker_pull_retry fluent/fluentd:v1.2-debian
+docker_pull_retry fluent/fluentd-kubernetes-daemonset:v1.2-debian-syslog
+
+# Pre pull images used in Ceph CSI
+docker_pull_retry quay.io/k8scsi/csi-attacher:v1.0.1
+docker_pull_retry quay.io/k8scsi/csi-provisioner:v1.0.1
+docker_pull_retry quay.io/k8scsi/csi-snapshotter:v1.0.1
+docker_pull_retry quay.io/cephcsi/rbdplugin:v1.0.0
+docker_pull_retry quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
+
+# Pre pull cluster network addons operator images and store manifests
+# so we can use them at cluster-up
+cp -rf /tmp/cnao/ /opt/
+for i in $(grep -A 2 "IMAGE" /opt/cnao/operator.yaml | grep value | awk '{print $2}'); do docker_pull_retry $i; done
+
+# Pre pull local-volume-provisioner
+for i in $(grep -A 2 "IMAGE" /provision/local-volume.yaml | grep value | awk -F\" '{print $2}'); do docker_pull_retry $i; done
+
+# Create a properly labelled tmp directory for testing
+mkdir -p /provision/kubevirt.io/tests
```

Comment:
see https://github.com/kubevirt/kubevirtci/pull/436 please
those line should be adapted as well

Reply:
Good point, will update.